2.4-micro/Dockerfile.fedora

```diff
@@ -0,0 +1,80 @@
+FROM quay.io/fedora/fedora:rawhide AS build
```

Comment:
Do we really want to use rawhide? I would prefer to use F36 or new but not rawhide? Do you have any reason why to use rawhide?

Reply:
Yes. Last I checked the httpd minimization work is still available only in rawhide